controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -217,7 +226,25 @@ function _sendReviewMailToAuthor($submission, $emailKey, $request) {
 
 		if(is_a($reviewRound, 'ReviewRound')) {
 			// Retrieve review indexes.
-			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /* @var $reviewAssignmentDao ReviewAssignmentDAO */
+			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /** @var ReviewAssignmentDAO $reviewAssignmentDao */
+
+			$reviewAssignments = $reviewAssignmentDao->getBySubmissionId($submission->getId(), $reviewRound->getId());
+			$reviewers = [];
+			/** @var ReviewAssignment $reviewAssignment */
+			foreach ($reviewAssignments as $reviewAssignment) {
+				if (in_array($reviewAssignment->getStatus(), [REVIEW_ASSIGNMENT_STATUS_COMPLETE, REVIEW_ASSIGNMENT_STATUS_RECEIVED, REVIEW_ASSIGNMENT_STATUS_THANKED])) {
+					$reviewers[$reviewAssignment->getReviewerId()] = 0;
+				}
+			}
+
+			$userDao = DAORegistry::getDAO('UserDAO'); /** @var UserDAO $userDao */
+			foreach (array_intersect(array_keys($reviewers), (array) $this->getData('bccReviewers')) as $reviewerId) {
+				$user = $userDao->getById($reviewerId);
+				if ($user) {
```

Comment:
Should probably check if the user is disabled, and not send the email in that case. `$user->getDisabled()`